### ISSUE TITLE
CuDNN error 

### ISSUE THREAD

**Issue by forslef** [kimjammer/Neuro#19]
error looks like this: Could not locate cudnn_ops64_9.dll. Please make sure it is in your library path!
I downloaded CuDNN 8.7.0, checked the files and there is indeed no such file there.
I have cuda version 11.8.
Is it possible that one of the libraries has been updated and requires a different version of cuda and therefore a different version of cuDNN?
The LLM loads normally, but the error occurs only at the moment when the STT starts working, i.e. when I start speaking
I am inexperienced in such matters, so if I ask something stupid I apologize.

**Comment by kimjammer** [comment]
Hi there! It's not a dumb question, library problems are common in ml land. I think you have the right version of CUDA and CuDNN, so here is what I would try. In a separate environment, try installing KoljaB/RealtimeSTT and run one of their example programs to make sure its working. If that works, then try using the pipfreeze.txt to make sure we're using the same version of every library.

**Comment by forslef** [comment]
I created a separate environment where I installed RealTimeSTT, torch and torchaudio version 2.5.1 is installed by default, and so everything works, but when I install version 2.5.1+cu118, the error appears, but asks for another file: **ERROR:root:General error in transcription: Library cublas64_12.dll is not found or cannot be loaded**
And with the version required by the project 2.2.2+cu118 the same error: **Could not locate cudnn_ops64_9.dll. Please make sure it is in your library path!**

I'm still pretty sure I have cuda 11.8 and cuDNN 8.7.0, and it seems to fit the requirements, so I have absolutely no idea what the cause could be. I also thought I might have specified something wrong in the environment variable, but that seems to be correct, but I'll attach a picture just in case.
![Uploading SystemPropertiesAdvanced_bnXG9kwwQb.png…]()

**Comment by Nenesh** [comment]
The "Could not locate cudnn_ops64_9.dll" came from faster-whisper and torch 2.2.2+cu118 . You need to downgrade ctranslate2 to 4.4.0 `pip install ctranslate2==4.4.0`

**Comment by jhj0517** [comment]
The `realtimestt` package depends on `faster-whisper==1.0.3`  which depends on `ctranslate2==4.5.0`. 
It's recommended to use `faster-whisepr==1.0.3` with CUDA 12.4.

If you have to use CUDA 11.8, just downgrade to `ctranslate2==4.4.0` as @Nenesh said.

**Comment by forslef** [comment]
Thank you all very much! After I did pip install ctranslate2==4.4.0, an error appeared because of faster-whisper,
I used the solution from here: https://github.com/jhj0517/Whisper-WebUI/issues/102#issuecomment-1974329102
But faster-whisper of a different version installs tokenizer of a different version, so I had to reinstall it.
To summarize the procedure to solve this error:
pip install ctranslate2==4.4.0
pip install faster-whisper==0.10.1
pip install tokenizers==0.19.1

Thank you all!
